lib/SILOptimizer/SILCombiner/SILCombiner.h

```diff
@@ -286,11 +286,18 @@ class SILCombiner :
                                               const ConcreteExistentialInfo &CEI,
                                               SILBuilderContext &BuilderCtx);
 
-  SILInstruction *
+  bool
```

Comment:
What does the return value mean?
(Please don't answer me in the PR. That's what function comments are for. )

Reply:
ok.